content/learn/book/getting-started/ecs/_index.md

```diff
@@ -175,7 +175,7 @@ fn main() {
 }
 ```
 
-Note that we have used `.chain()` on the two systems. This is because we want them two to run in exactly the order they're listed in the code: with `update_people` occurring before `greet_people`.
+Note that we have used `.chain()` on the two systems. This is because we want them to to run in exactly the order they're listed in the code: with `update_people` occurring before `greet_people`.
```

Comment:
```suggestion
Note that we have used `.chain()` on the two systems. This is because we want both of them to run in exactly the order they're listed in the code: with `update_people` occurring before `greet_people`.
```

Reply:
I definitely think there's a problem with the current phrasing, but I think this is the right fix.